BrainPortal/lib/cbrain_task_generators/templates/bourreau.rb.erb

```diff
@@ -363,22 +363,24 @@ class CbrainTask::<%= name %> < <%= (descriptor['custom'] || {})['cbrain:inherit
 % unless (descriptor['custom'] || {})['cbrain:ignore-exit-status']
     # Make sure <%= name %> completed successfully by checking its exit status
     # in +exit_cluster_filename+.
-    unless File.exists?(exit_cluster_filename)
-      self.addlog("Missing exit status file #{exit_cluster_filename}")
+    if File.exists?(exit_cluster_filename)
+      exit_status = IO.read(exit_cluster_filename).strip.to_i
+      unless SystemExit.new(exit_status).success?
```

Comment:
Not sure why we build and entire SystemExit exception object just to test success... seems a bit overkill.